### ISSUE TITLE
Replace NAMESERVER:NS_ERROR

### ISSUE THREAD

**Issue by mattias-p** [zonemaster/zonemaster#755]
The message NAMESERVER:NS_ERROR isn't actionable from an end user perspective. It is emitted in various circumstances. For every circumstance in which it is emitted, we should replace it with a more pointed message that lets the end user troubleshoot the name server using simpler tools.

**Comment by matsduf** [comment]
I agree with the criticism above. See issue zonemaster/zonemaster-engine#529 for another solution.

**Comment by matsduf** [comment]
If we follow the proposal in zonemaster/zonemaster-engine#578 the we can solve this issue.
